### DISCUSSION TITLE
Error at start:  `horde_worker_regen.download_models:download_all_models:33 - 'reGenBridgeData' object has no attribute 'meta_skip_instructions'`

### DISCUSSION THREAD

**Discussion by martinellison** [Haidra-Org/AI-Horde#385] [Q&A] [answered]
As indicated, when I run `horde-bridge.sh` I get this message "`Error at start:  `horde_worker_regen.download_models:download_all_models:33 - 'reGenBridgeData' object has no attribute 'meta_skip_instructions'``".

This was working recently.

This is on Fedora Linux 39 with an RTX 3060. Worker is up to date on main.

My `bridgedata.yaml` currently includes:
```
models_to_load:
  - "TOP 10"
```

Any ideas as to what is going wrong and how to fix it?

**Comment by tazlin** [comment, marked as answer]
Sorry you encountered this issue.

There has been a flurry of updates in different places the past few days; my guess this is related to that.

You can try the following in the worker directory:
```bash
git remote prune origin && git pull
./update-runtime.sh
./horde-bridge.sh
```

If this issue isn't resolved by doing this, please me know.

**Reply by martinellison** [reply]
This worked thanks. Have generated 15 kudos now...
